src/main/java/org/alfresco/heartbeat/InfoDataCollector.java

```diff
@@ -52,16 +65,34 @@
  *                  <li>minor: String - The minor version number, e.g. 1.<u>2</u>.3. {@link Descriptor#getVersionMinor()}</li>
  *                  <li>patch: String - The version revision number, e.g. 1.2.<u>3</u>. {@link Descriptor#getVersionRevision()}</li>
  *                  <li>hotfix: String - The version label. {@link Descriptor#getVersionLabel()}</li>
+ *                  <li>build: String - The number of the build. {@link Descriptor#getVersionBuild()}</li>
  *              </ul>
  *          </li>
  *          <li><b>schema:</b> Int - The schema number. {@link Descriptor#getSchema()}</li>
  *          <li><b>edition:</b> String - The edition. {@link Descriptor#getEdition()}</li>
  *          <li><b>deploymentMethod:</b> String - The deployment method used to deploy this Alfresco instance. {@link DeploymentMethodProvider#getDeploymentMethod()}</li>
+ *          <li><b>os.vendor:</b> String - The name of the Operating System vendor. {@link System#getProperty(String)}</li>
+ *          <li><b>os.version:</b> String - The version of the Operating System. {@link System#getProperty(String)}</li>
+ *          <li><b>os.arch:</b> String - The architecture of the Operating System. {@link System#getProperty(String)}</li>
+ *          <li><b>java.vendor:</b> String - The name of the Java vendor. {@link System#getProperty(String)}</li>
+ *          <li><b>java.version:</b> String - The version of Java used. {@link System#getProperty(String)}</li>
+ *          <li><b>user.language:</b> String - The language which this instance was installed with. {@link Locale#getLanguage()} </li>
```

Comment:
As a good practice in Elasticsearch (where this data gets) `"."` is not encouraged to be part of a field name, since it is a special character used for navigating into components.

Reply:
Changes in the next commit.